templates/kafka/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Apache Kafka
+description: |
+  (Experimental) Kafka cluster
+version: 0.8.2-rancher1
+category: clustering
+release by: Raul Sanchez
```

Comment:
Done the changes, but drone build fails. Is this issue about the changes or about the system??
The package is working well in my rancher.